### DISCUSSION TITLE
ipxe iso bootable issue using https

### DISCUSSION THREAD

**Discussion by amardeepl11** [ipxe/ipxe#1154] [Q&A] [answered]
Hello All,

I am trying to create a ipxe.iso file to boot in clonezilla .
clonezilla files are saved at https server, https server has root ca, intermediate ca and server certificate and is showing secure in browser.
Now when i have created iso using below commands I am getting below error.
make bin/ipxe.lkrn bin-x86_64-efi/ipxe.efi TRUST=/root/ca/server/cert.pem EMBED=boot.ipxe
./util/genfsimg -o ipxe.iso bin/ipxe.lkrn bin-x86_64-efi/ipxe.efi

Note - TRUST=/root/ca/server/cert.pem is server certificate not root CA certificate.

![image](https://github.com/ipxe/ipxe/assets/139690905/86d6197a-0f54-4337-8a70-c40e6e241428)

**Comment by amardeepl11** [comment]
![image](https://github.com/ipxe/ipxe/assets/139690905/71b75b4f-52c2-4df2-8551-06057e09031c)

**Comment by NiKiZe** [comment, marked as answer]
You are getting an error from wget, in clonezilla, not from iPXE.
It is very possible that the basic environment in the initrd does not have https support.

Check with clonezilla.

**Reply by amardeepl11** [reply]
Hi @NiKiZe ,

Thanks , but i can see when i am not using https, it works. it seems there the issue is with ipxe iso which can not access filesquashfs file on https.

**Reply by amardeepl11** [reply]
![image](https://github.com/ipxe/ipxe/assets/139690905/975bf5b1-41e1-48c0-b742-e02ba37a9e5d)

**Reply by mcb30** [reply]
@amardeepl11 All of your error messages are coming from within the loaded operating system, not from iPXE.  It's the loaded operating system (specifically, the code running in your initramfs) which is attempting and failing to perform an HTTPS download.

**Reply by NiKiZe** [reply]
> Thanks , but i can see when i am not using https, it works.

Exactly, because wget in clonezilla does not support https.

How does your iPXE script look?
